docs/testnet/onomy-testnet-docs/faucet.md

```diff
@@ -0,0 +1,105 @@
+# Onomy testnet faucet
+
+In order to use the faucet and receive testnet tokens you'll first need to setup a wallet
+
+## What do I need?
+
+A Linux server with any modern Linux distribution, 2gb of ram and at least 20gb storage. Requirements are very minimal.
+
+### Download/install Onomy chain binaries
+```
+To download binary follow these commands
+mkdir binaries
+cd binaries
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/onomyd
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/gbt
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/geth
+cd ..
+chmod -R +x binaries
+export PATH=$PATH:$HOME/binaries/
+
+
+or If you have Fedora (Fedora 34) or Redhat (Red Hat Enterprise Linux 8.4 (Ootpa))
+ and you want to make binaries yourself, then follow these steps
+
+sudo yum install -y git
+git clone -b dev https://github.com/onomyprotocol/onomy.git
+cd onomy/deploy/testnet
+bash bin.sh
+```
+### Init the config files
+
+```
+cd $HOME
+onomyd --home $HOME/onomy/onomy init mymoniker --chain-id onomy
```

Comment:
`onomyd --home $HOME/onomy/onomy init {validator moniker} --chain-id onomy`

I'd recommend this instead so people aren't just copy/pasting and make a mistake.

Reply:
@sunnyk56 check the comments please

Reply:
all above suggestion has been implemented